symfony/web-profiler-bundle/6.1/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+    "bundles": {
+        "Symfony\\Bundle\\WebProfilerBundle\\WebProfilerBundle": ["dev", "test"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "conflict": {
+        "symfony/framework-bundle": "<5.3"
```

Comment:
should be 6.1, isn't it?

Reply:
probably yes since the option is from framework-bundle, what made you rollback? ^^